atest/testdata/standard_libraries/builtin/verify.robot

```diff
@@ -68,6 +68,11 @@ Should Not Be Equal
     ${STR1}    ${INT1}
     ${STR1}    1
 
+Should Not Be Equal With Case Insensitivity
+    [Template]  Should Not Be Equal
+    test value      TEST VALUE1      ignore_case=True
+    HYVÄÄ YÖTÄ      hyvää yötä1     ignore_case=True
```

Comment:
I think I see what you're getting at, but not sure.  If I modify the test case to look at the case: 
   ...
    foo                      FOO                ignore_case=True

then ignoring case would make the strings equal, which would in-turn fail this Acceptance Test.

Putting the extra character on the second argument was the best that I could think of, since just modifying case isn't enough to keep the strings from being unequal.
Am I missing something right in front of my nose here?